include/pstore/mcrepo/compilation.hpp

```diff
@@ -63,6 +63,17 @@ namespace pstore {
 
         std::ostream & operator<< (std::ostream & os, linkage_type l);
 
+#define PSTORE_REPO_VISIBILITY_TYPES                                                               \
+    X (default)                                                                                    \
+    X (hidden)                                                                                     \
+    X (protected)
+
+#define X(a) a##_visibility,
```

Comment:
This _visibility suffix appears more than once. Presumably its there because "default" is a keyword. Could we use some other term to avoid the suffix?

Reply:
> 
> 
> Presumably its there because "default" is a keyword. Could we use some other term to avoid the suffix?
>
>
Yes, both  "default"  and "protected" are keywords. Both these two types need a suffix.

I borrowed enum types from GlobalValue:
```
  enum VisibilityTypes {
    DefaultVisibility = 0,  ///< The GV is visible
    HiddenVisibility,       ///< The GV is hidden
    ProtectedVisibility     ///< The GV is protected
  };
```
Please give me some advice if they need to be changed to some other terms.